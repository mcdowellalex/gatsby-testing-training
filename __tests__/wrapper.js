// testing css in js through a theme provider

import React from "react"
import { ThemeProvider, withTheme } from "@emotion/react"
import renderer from "react-test-renderer"
import styled from 'styled-components';

const theme = {
  maxWidth: "1450px",
}

const Wrapper = styled.section`
  max-width: ${props => props.theme.maxWidth};
`

test("Wrapper renders correctly", () => {
  expect(
    renderer
      .create(
        <ThemeProvider theme={theme}>
          <Wrapper>Content.</Wrapper>
        </ThemeProvider>
      )
      .toJSON()
  ).toMatchSnapshot()
})