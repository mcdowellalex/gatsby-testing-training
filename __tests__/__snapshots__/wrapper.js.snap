// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper renders correctly 1`] = `
<section
  className="sc-bdvvtL bdgLDQ"
>
  Content.
</section>
`;
